### ISSUE TITLE
Sound notification when player joins

### ISSUE THREAD

**Issue by woothie** [HeavensGate/Eternal#30]
Add a toggable sound for when a player joins the round for admins. Further, add a text announcement for when a player joins the server for all to see. Discuss the last point, not sure if Core approves of world announcement on join. Regardless, add player joining to adminlog and debug.

**Comment by woothie** [comment]
Half-way done, admins now get a text notification for when a client connects. Now to find out how to play a sound for select clients (read: admins) and make it toggable.

**Comment by Davnik** [comment]
I don't think we need to globally announce to players when a client joins the server. I do think, however, that admins would like that advance warning. All the better to have those controls in case we want them, on toggle if we don't.

Also, a second thought: Perhaps work on a feature to gather statistics on someone's most frequently played one or two characters, compiled from how much time in-rounds they've played them. sometimes it'll be 13/12/10/10/10/10/10/9/8/8, sometimes it'll be 90/10, and probably a lot of 30/30/20/8/6/3/2/1. But having the one or two "most played" would be useful.

**Comment by Mewchild** [comment]
I feel we should be able to tage specific ckeys for anotifcation when ONLY THE TAGGED KEYS join, so we can watch for troublemakers

**Comment by Davnik** [comment]
Agreed, tagging ckeys sounds good.

I'm trying to think of how to word something to add to that thought, but it's coming out wrong mentally. Just... tagging ckeys to cut down on all the chatter so admins pay attention to the client join notifications is good.
